src/github.go

```diff
@@ -133,7 +133,7 @@ func auditGithubRepos() ([]Leak, error) {
 			break
 		}
 		if opts.GithubUser != "" {
-			pagedGithubRepos, resp, err = githubClient.Repositories.List(ctx, opts.GithubUser, githubOptions)
+			pagedGithubRepos, resp, err = githubClient.Repositories.List(ctx, "", githubOptions)
```

Comment:
I think this logic needs some additional work. What about the case when a user is attempting to audit a some user's public repos?

Reply:
@zricethezav thanks for sharing your comments, and I totally agree with you.

I just added the logic that the tool would scan both public and private repo for the authenticated user but only public repo for another user.

Reply:
@zricethezav could you pls help review my changes? many thanks.